client/src/get-compiled-template.js

```diff
@@ -32,11 +23,20 @@ module.exports = function(cache){
             }
           });
         }
-        var type = template.type;
-        if (type === 'jade') { type = 'oc-template-jade'; }
-        if (type === 'handlebars') { type = 'oc-template-handlebars'; }
 
-        cb(null, templateEngines[type].getCompiledTemplate(templateText, template.key));
+        var type = template.type;
+        var ocTemplate;
+        try {
+          if (type === 'jade') { type = 'oc-template-jade'; }
+          if (type === 'handlebars') { type = 'oc-template-handlebars'; }
+
+          // dynamically require specific oc-template
+          ocTemplate = require(type);
+        } catch (err) {
+          throw format(settings.templateNotSupported, type);
+        }
+        
+        cb(null, ocTemplate.getCompiledTemplate(templateText, template.key));
```

Comment:
I am thinking that these 5 lines of code are actually repeated 4 times. Given we need them for both npm modules (oc and oc-client) - what about moving it to its own npm module, like `oc-require-templates` so that we can just require it all the times? If that doesn't require too much time obviously?
If not, I would just move it to its own file, like `oc-require-templates.js` and duplicate it inside the /client and /src folders.